packages/theme/template/ThemeDecorator/ThemeDecorator.js

```diff
@@ -5,16 +5,17 @@
  */
 
 import {addAll} from '@enact/core/keymap';
+import kind from '@enact/core/kind';
 import hoc from '@enact/core/hoc';
 import I18nDecorator from '@enact/i18n/I18nDecorator';
-import kind from '@enact/core/kind';
-import {ResolutionDecorator} from '@enact/ui/resolution';
 import {FloatingLayerDecorator} from '@enact/ui/FloatingLayer';
+import {ResolutionDecorator} from '@enact/ui/resolution';
 import SpotlightRootDecorator from '@enact/spotlight/SpotlightRootDecorator';
 
 import Skinnable from '../Skinnable';
 
 import screenTypes from './screenTypes.json';
+
```

Comment:
I don't think this empty line si necessary here.

Reply:
They are different import types. package imports vs css. Check the guide